docs/features.rst

```diff
@@ -1,70 +1,91 @@
 Read the Docs features
 ======================
 
-This will serve as a list of all of the features that Read the Docs currently has. Some features are important enough to have their own page in the docs, others will simply be listed here.
+Read the Docs offers a number of platform features that are possible because we both build and host documentation for you.
 
-GitHub, Bitbucket and GitLab Integration
-----------------------------------------
 
-We now support linking by default in the sidebar. It links to the page on your host, which should help people quickly update typos and send pull requests to contribute to project documentation.
+Automatic Documentation Deployment
+----------------------------------
 
-More information can be found in :doc:`guides/vcs`.
+We integrate with GitHub, BitBucket, and GitLab.
+We automatically create webhooks in your repository,
+which tell us whenever you push a commit.
+We will then build and deploy your docs every time you push a commit.
+This enables a workflow that we call *Continuous Documentation*:
 
-Auto-updating
--------------
+**Once you set up your Read the Docs project,
+your users will always have up-to-date documentation.**
 
-The :doc:`webhooks` page talks about the different ways you can ping RTD to let us know your project has been updated. We have official support for GitHub, Bitbucket and GitLab, and anywhere else we have a generic post-commit hook that allows you to POST to a URL to get your documentation built.
+Learn more about :doc:`/webhooks`.
 
-Internationalization
---------------------
+Custom Domains & White Labeling
+-------------------------------
 
-Read the Docs itself is localized, and we support documentation translated into multiple languages.
-Read more on the :doc:`localization` and :doc:`development/i18n` pages.
+When you import a project to Read the Docs,
+we assign you a URL based on your project name.
+You are welcome to use this URL,
+but we also fully support custom domains (``CNAME's``) for all our documentation projects.
```

Comment:
Maybe not in this PR but getting rid of the term CNAME (unless referring specifically to DNS) is probably good. We should call them "custom domains". This is especially true now that we explicitly support root domains which are not CNAMEs.

Reply:
Sure, I just want it to be explicit and searchable. It's called custom domains in the primary content, but I think its useful to mention.

Reply:
I'm +1 on removing CNAME here as well, this usage is confusing as it's a DNS term.